README.md

```diff
@@ -54,11 +54,20 @@ configuration.setParallelTesting(parallelTesting);
 configuration.setRunWithJenkins(runWithJenkins);
 configuration.setBuildNumber(buildNumber);
 // addidtional metadata presented on main page
-configuration.addClassifications("Platform", "Windows");
-configuration.addClassifications("Browser", "Firefox");
 configuration.addClassifications("Branch", "release/1.0");
+configuration.addClassifications("Browser", "Firefox");
+configuration.addClassifications("Platform", "Windows");
 
-ReportBuilder reportBuilder = new ReportBuilder(jsonFiles, configuration);
+List<String> propertiesFiles = new ArrayList<>();
+// properties-1 contains...
+// AutUiVersion=1.25.3
+// AutApiVersion=1.25.0.7
```

Comment:
remove those comments - I don't think this is valuable for readme. Just put small piece of code to expose that properties are supported now

Reply:
yeah I'll remove them in updated PR...